src/index.ts

```diff
@@ -48,21 +54,23 @@ const UI = {
   create: require('./create'),
   icons: require('./iconBase'),
   log: require('./log'),
-  matrix: require('./matrix'),
+  matrix: matrix,
```

Comment:
Maybe just use the short hand form, to clean up the code up a bit more?

Reply:
I'm generally not a fan of mixing the shorthand and regular form (or of using the shorthand form outside of one-liners in general), but I don't feel strongly about it so if you do - sure :) In that case, might also be a good idea to add an ESLint rule?